manager/state/raft/raft.go

```diff
@@ -1142,15 +1145,30 @@ func (n *Node) send(ctx context.Context, messages []raftpb.Message) error {
 			continue
 		}
 
+		ch := make(chan struct{})
```

Comment:
Also, the cost of creating a channel should be much less than the cost of doing RPC over the network.